src/app/danger-message/danger-message.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'danger-message',
+  templateUrl: './danger-message.component.html',
+  styleUrls: ['./danger-message.component.css']
+})
+export class DangerMessageComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+  IsHidden= true;
+  onSelect(){
+    this.IsHidden= !this.IsHidden;
```

Comment:
this is good. its good that you implemented something that wasn't asked for.

Reply:
is this part of the additional challenge? if so then it is implemented wrong. you need to add a button to appComponent which selects which message to display. 

Regardless of whether this was an attempt at the additional challenge, it works and good job for that.